docker/hyproof-apis.yml

```diff
@@ -19,11 +19,11 @@ services:
 
   hydrogen-producer-hyproof-api:
     container_name: hydrogen-producer-hyproof-api
-    image: digicatapult/sqnc-hyproof-api:latest
+    image: digicatapult/sqnc-hyproof-api:v0.13.25
```

Comment:
check all current versions just before final merge

Reply:
DONE.

Reply:
Hmmm , actually I need to deactivate renovate first which is this PR here:

https://github.com/digicatapult/sqnc-hyproof-client/pull/148

Reply:
> check all current versions just before final merge

Just finished doing another check ..